HANDOFF.md

```
@@ -0,0 +1,107 @@
+- [ ] **Performance tests** — No benchmarks for timeline rendering, WASM compositor, or large project handling.
+
+## Technical Notes
+
+- **Test runner**: `bun test` (bun@1.3.14+)
+- **Install deps first**: `cd /tmp/OpenCut && bun install`
+- **Run all new tests**: `bun test apps/web/src/subtitles/__tests__/ apps/web/src/utils/__tests__/math-extended.test.ts apps/web/src/fps/__tests__/fps-extended.test.ts apps/web/src/rendering/__tests__/ apps/web/src/params/__tests__/params.test.ts`
+- **Run single file**: `bun test apps/web/src/subtitles/__tests__/srt.test.ts`
+
+## Files Created
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)